9/Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM ubuntu:16.04
 
-ADD config/redis.config /etc/redis/redis.config
-ADD start /start
+COPY config/redis.config /etc/redis/redis.config
```

Comment:
@mikesplain do you really want all COPY files up here... it means blowing away all the cache layers every time a change is created. And the build is quite long already.